acceptance/bundle/resources/experiments/change-artifact-loc/out.test.toml

```diff
@@ -3,3 +3,4 @@ Cloud = true
 
 [EnvMatrix]
   DATABRICKS_BUNDLE_ENGINE = ["terraform", "direct-exp"]
+  Ignore = [".databricks"]
```

Comment:
It looks like you wanted to use top level Ignore setting, but nested it under EnvMatrix.